doc/changes.md

```diff
@@ -1,5 +1,113 @@
 # Changelog
 
+## Release v0.14.0 (2024/12/31)
+
+[![Digital Object Identifier for PyGMT v0.14.0](https://zenodo.org/badge/DOI/10.5281/zenodo.14535921.svg)](https://doi.org/10.5281/zenodo.14535921)
+
+### Highlights
+
+*
+
+### New Features
+
+* Add Figure.hlines for plotting horizontal lines ([#923](https://github.com/GenericMappingTools/pygmt/pull/923))
+* Add Figure.vlines for plotting vertical lines ([#3726](https://github.com/GenericMappingTools/pygmt/pull/3726))
+* Add "load_blue_marble" to load Blue Marble dataset ([#2235](https://github.com/GenericMappingTools/pygmt/pull/2235))
+* Add "load_black_marble" to load Black Marble dataset ([#3469](https://github.com/GenericMappingTools/pygmt/pull/3469))
+* Add "load_earth_mean_sea_surface" to load "CNES Earth Mean Sea Surface" dataset ([#3717](https://github.com/GenericMappingTools/pygmt/pull/3717))
+* Add "load_earth_mean_dynamic_topography" to load "CNES Earth Mean Dynamic Topography" dataset ([#3718](https://github.com/GenericMappingTools/pygmt/pull/3718))
+* Add "load_earth_dist" to load "GSHHG Earth distance to shoreline" dataset ([#3706](https://github.com/GenericMappingTools/pygmt/pull/3706))
```

Comment:
Are the double quotes `" "` needed? In PyGMT v0.8.0 (https://www.pygmt.org/v0.13.0/changes.html#id31), we just had entries like `Add load_earth_geoid function for Earth Geoid dataset`

Reply:
Removed in 5e3cc60c5.